lib/commits.js

```diff
@@ -35,6 +35,7 @@ module.exports.findCommitsWithAssociatedPullRequestsQuery = /* GraphQL */ `
                   nodes {
                     title
                     number
+                    url
```

Comment:
could we do the same include trick as we did for body? Again to avoid pulling extra data that is not needed.

Reply:
It makes sense

Reply:
Done